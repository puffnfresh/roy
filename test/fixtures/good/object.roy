console.log {}
console.log {a: 1}
console.log {'b': 10}
console.log {"c": 11}
console.log {4: 100}

let x = {}
console.log x

let y = {'test': 'this'}
console.log y
