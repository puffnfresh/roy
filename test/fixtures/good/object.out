{}
{ a: 1 }
{ b: 10 }
{ c: 11 }
{ '4': 100 }
{}
{ test: 'this' }
